index.html

```diff
@@ -274,8 +275,16 @@
         row.style.left = modeDivPosition.left + 'px';
         row.style.top = modeDivPosition.top + 'px';
     };
-  </script>
 
+    function playSound(soundObj) {
+        var sound = document.getElementById(soundObj);
+        sound.play();
```

Comment:
If we create an object instance just to call it function, maybe it is easier to do it in one line? `docById('fdsfdf').play()`